config/routes.rb

```diff
@@ -74,7 +74,10 @@
   get "/auth/failure", to: "volunteer/sessions#failure"
 
   namespace :admin do
-    resources :organizations
+    resources :organizations do
+      resources :organization_members, only: [:new, :create]
+    end
+    resources :organization_members, only: [:show, :edit, :update, :destroy]
```

Comment:
Two sets of resources:

- one that includes the `organization_id` in the url
- one that contains the `organization_member_id`